packages/client/src/components/Component.svelte

```diff
@@ -368,6 +374,9 @@
     }
   }
 
+  // Check for invalid settings
+  $: invalidSettings = $componentErrors[id]
```

Comment:
Could we hoist this up a bit to be beside the other reactive statements? This component is a monster and I find it much easier to read with all the reactive logic in one place 🙏